### PR TITLE
Disable IPv6 in Clickhouse config

### PR DESCRIPTION
**Pull request by rmens**
To suppress this warnings https://github.com/plausible/analytics/discussions/4232

### CODE REVIEW

File: clickhouse/clickhouse-config.xml [hunk 1]
```diff
@@ -3,6 +3,9 @@
         <level>warning</level>
         <console>true</console>
     </logger>
+    
+    <!-- Don't listen on IPv6 since it's not avaiable in Docker -->
+    <listen_host>0.0.0.0</listen_host>
```
Comment (ruslandoga) [inline clickhouse/clickhouse-config.xml:11]: It is available if people decide to enable it :)

Comment (ruslandoga) [reply clickhouse/clickhouse-config.xml:11]: I meant that we shouldn't disable it by default. We can document it though and keep this line commented out.

Comment (rmens) [reply clickhouse/clickhouse-config.xml:11]: Added some inline documentation. Feel free to modify as needed.

Comment (ruslandoga) [reply clickhouse/clickhouse-config.xml:11]: Thank you!

**Pull request merged**